resolwe_bio/processes/scseq/cellranger.py

```diff
@@ -264,15 +265,21 @@ def run(self, inputs, outputs):
         }
         self.run_process('upload-bam-scseq-indexed', process_inputs)
 
-        filtered_dir = os.path.join(output_dir, 'filtered_feature_bc_matrix')
-        raw_dir = os.path.join(output_dir, 'raw_feature_bc_matrix')
+        report_file = '{}_summary.html'.format(sample_name)
```

Comment:
This may cause issues if users change the sample name later. How about simply `report_summary.html`?

Reply:
This is true. When deciding on using a sample name I was thinking of `ReSDK` download function. Say we have many scRNA samples and we want to download all the reports using ReSDK. If the reports have the same name, each time we call the download function the previously downloaded report will get overwritten. Now that I think about it more I guess I tried to solved the issue in the wrong place.